RetroBar/Controls/TaskButton.xaml.cs

```diff
@@ -180,6 +181,33 @@ private void CloseMenuItem_OnClick(object sender, RoutedEventArgs e)
             Window?.Close();
         }
 
+        private void EndTaskMenuItem_OnClick(object sender, RoutedEventArgs e)
+        {
+            if (Window != null)
+            {
+                ForceEndTask();
+            }
+        }
+
+        private void ForceEndTask()
+        {
+            try
+            {
+                uint processId;
+                NativeMethods.GetWindowThreadProcessId(Window.Handle, out processId);
+                
+                if (processId != 0)
```

Comment:
I think there should also be a check if this is the RetroBar PID (i.e. in the case of the Properties window). In that case, use `Window?.Close()` instead.